docs/UserGuide.md

```diff
@@ -342,9 +357,31 @@ Exits the program.
 
 Format: `exit`
 
-### 6.11. Viewing daily tips
+### 6.11. Viewing application stage tips
+
+For some specific application stages, we have included a list of useful tips to help you prepare for and ace the deliverables.
+
+Below is a list of application stages that we have tips for.
+
+| Application Stage |
+|--------|
+| `Application Sent` |
+| `Online Assessment` |
+| `Technical Interview` |
+| `Behavioral Interview` |
+| `Phone Interview` |
+| `Rejected` |
+
+To view the tips in the narrow Ui format, click the light bulb button on the application card. A window will appear displaying tips for the stage that the application is currently at.
```

Comment:
Perhaps can shorten and use bold or italics for contrast? something like
```suggestion
If WorkBook is in the **narrow layout** (hyperlink the bolded words to the explanation about narrow layout above), simply click on the light bulb icon to view the tips for that application.
```

Reply:
I've added the anchor links.